### ISSUE TITLE
experiment results: separate networks or unified?

### ISSUE THREAD

**Issue by RuyiLian** [suyz526/ZebraPose#1]
Hi,

I have read your paper and the idea is really interesting! But I am not sure whether you train separate networks for each object (i.e. `P.E.=N` defined in [GDR-Net](https://arxiv.org/pdf/2102.12145.pdf)) or train one unified network for all objects (i.e. `P.E.=1`). Could you help to clarify this? Thanks!

Also, in my understanding, the results of GDR-Net are `P.E.=N` while those of SO-Pose are `P.E.=1`.Since `P.E.=N` tends to outperform `P.E.=1` for the same method, putting these results together without additional remarks may look a bit weird.

**Comment by suyz526** [comment]
Hi,

thanks for the question. We trained a separate network for each object.

The approaches vary a lot, some trained separately, while some trained unified; some are trained with synthetic images, some are trained with PBR images; some directly regress the pose, some are refinement-based, etc...

We have considered putting all the details, but it will make the table a bit hard to read and take a lot of space. So in the end we just selected all the best-reported results from the most recent research papers. We will report more results in the BOP challenge, there you can compare our method with others, which are trained in the same rules.

**Comment by RuyiLian** [comment]
Thanks for your detailed reply! I am looking forward to your codes and more BOP results. In my understanding, for BOP challenge, you also train a separate network for each object, but using only PBR images. Please correct me if I make any mistake. Thanks!

**Comment by suyz526** [comment]
Yes, we trained 1 CNN for 1 object in the BOP challenge.

When we submit the results to the leader board, we are asked to select the training image set. If you open the submission of each dataset, there will be information about the training data, e.g. for TUD-L:
[https://bop.felk.cvut.cz/sub_info/2117/](https://bop.felk.cvut.cz/sub_info/2117/)

In our submissions, the 'synthetic' always means PBR images.

For BOP, the LM-O dataset is not allowed to be trained with PBR+real, so we trained only with PBR images. (For the results in the paper, we used pbr+real, as GDR-Net did). For all other datasets in BOP, we used, and plan to use pbr+real images for training.
